libraries/Microsoft.Bot.Builder.Ai/QnAMakerMiddleware.cs

```diff
@@ -28,7 +42,13 @@ public QnAMakerMiddleware(QnAMakerOptions options, HttpClient httpClient)
                     var results = await _qnaMaker.GetAnswers(messageActivity.Text.Trim()).ConfigureAwait(false);
                     if (results.Any())
                     {
+                        if (!string.IsNullOrEmpty(_options.DefaultAnswerPrefixMessage))
+                            context.Reply(_options.DefaultAnswerPrefixMessage);
+
                         context.Reply(results.First().Answer);
+
+                        if (_options.EndActivityRoutingOnAnswer)
```

Comment:
Consider adding a comment here calling out "Question is answered, don't keep routing"

Reply:
Done.